dali_tf_plugin/dali_dataset_op.cc

```diff
@@ -41,6 +41,7 @@
 #include "tensorflow/core/framework/types.h"
 
 #include "tensorflow/core/framework/tensor.h"
+#include "tensorflow/core/framework/full_type_util.h"
```

Comment:
```suggestion
```suggestion,
#if TF_MAJOR_VERSION > 2 || (TF_MAJOR_VERSION == 2 && TF_MINOR_VERSION >= 8)
#include "tensorflow/core/framework/full_type_util.h"
#endif
```
```

Reply:
Right :man_facepalming: